src/hardware/joystick.cpp

```diff
@@ -534,8 +534,8 @@ static void configure_calibration(const Section_prop &settings)
 			return parsed_rates;
 		}
 		if (pref != "auto" && pref.length() != 0)
-			LOG_WARNING("JOYSTICK: Invalid %c_calibration parameters: %s."
-			            " Must be auto or number,number.",
+			LOG_WARNING("JOYSTICK: Invalid '%c_calibration' setting: '%s', "
+			            "using 'auto'",
```

Comment:
Why not notifying what the correct format is (number,number)? 
Further up a similar hint for a display setting is kept.

Reply:
I guess I can re-add.

Reply:
That message I removed was actually out of date. Simply going with my new version.